plugins/woocommerce/includes/data-stores/class-wc-customer-data-store-session.php

```diff
@@ -163,37 +200,4 @@ protected function set_defaults( &$customer ) {
 	public function delete( &$customer, $args = array() ) {
 		WC()->session->set( 'customer', null );
 	}
-
-	/**
-	 * Gets the customers last order.
-	 *
-	 * @since 3.0.0
-	 * @param WC_Customer $customer Customer object.
-	 * @return WC_Order|false
-	 */
-	public function get_last_order( &$customer ) {
```

Comment:
Extending `WC_Customer_Data_Store` means we don't need to worry about these methods, and as a bonus they'll work for logged in users.